app/src/main/res/layout/activity_all_events.xml

```diff
@@ -18,6 +19,7 @@
             android:id="@+id/event_back"
             android:layout_width="70dp"
             android:layout_height="70dp"
+            android:layout_weight="0.1"
```

Comment:
I think that this code is not responsive.
please check it on tablet and large phones then comment here.

Reply:
well i ckecked it again and it works fine for me!
okay  i 'll keep that in mind!

Reply:
Thank you!